src/cli/config-flags.ts

```diff
@@ -35,12 +35,65 @@ export const BOOLEAN_CLI_ARGS = [
   'verbose',
   'version',
   'watch',
+
+  // JEST CLI OPTIONS
```

Comment:
I separated these will a little comment heading for ease of editing later if / when Jest's supported arguments change

Reply:
please add `outputFile`